### PR TITLE
Add plugin helper

### PR DESCRIPTION
**Pull request by robinsoc**
This is a part of [request #15033](https://tuleap.net/plugins/tracker/?aid=15033) Improve the Jenkins Authentication plugin

In the Security Realm menu you should see a helper next to "Tuleap Authentication" radio button.

### CODE REVIEW

File: src/main/webapp/helpTuleapSecurityRealm/helpTuleapRealm/help_fr.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<div>
+    En utilisant se domaine, les utilisateurs Jenkins peuvent utiliser leur compte Tuleap pour se connecter au serveur
```
Comment (Erwyn) [inline src/main/webapp/helpTuleapSecurityRealm/helpTuleapRealm/help_fr.html:2]: [se][ce]

**Comment by Erwyn** [comment]
Functional ✅

File: src/main/webapp/helpTuleapSecurityRealm/helpTuleapRealm/help_fr.html (hunk 1 above)
Comment (robinsoc) [reply src/main/webapp/helpTuleapSecurityRealm/helpTuleapRealm/help_fr.html]: Thank you

File: src/main/webapp/helpTuleapSecurityRealm/helpTuleapRealm/help_fr.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<div>
+    En utilisant ce domaine, les utilisateurs Jenkins peuvent utiliser leur compte Tuleap pour se connecter au serveur
+    Jenkins. <br/>
+    Pour pouvoir être autentifié en utilisant Tuleap, vous devez rajouter l'URL de redirection du serveur Jenkins dans
```
Comment (Erwyn) [inline src/main/webapp/helpTuleapSecurityRealm/helpTuleapRealm/help_fr.html:4]: aut[][h]entifié

Comment (robinsoc) [reply src/main/webapp/helpTuleapSecurityRealm/helpTuleapRealm/help_fr.html:4]: :+1:

**Pull request merged**